lib/presentation/resources/todo_cards/todo_cards.screen.dart

```diff
@@ -0,0 +1,179 @@
+          Icons.calendar_month,
+        ),
+      ),
+    );
+  }
+
+  Widget _buildBody(List<TodoModel> todoCards) {
+    return LayoutBuilder(builder: (context, dimens) {
+      return ListView.builder(
+        itemCount: todoCards.length,
+        itemBuilder: (context, index) {
+          return _buildCard(todoCards[index]);
+        },
+      );
+    });
+  }
+
+  Widget _buildCard(TodoModel todoCard) {
+    double todoCardTextAreaWidth = getScreenWidth(context) * 0.7;
+    return Padding(
+      padding: const EdgeInsets.only(top: 20.0, left: 16.0, right: 16.0),
+      child: Dismissible(
+        key: const Key('card'), //TODO: replace with snapshot id
+        child: Card(
+            elevation: 2,
+            color: Colors.grey.shade100,
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Padding(
+                  padding:
+                      const EdgeInsets.only(top: 12.0, left: 20.0, right: 20.0),
+                  child: SizedBox(
+                    height: 100,
+                    width: todoCardTextAreaWidth,
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: [
+                        Text(todoCard.title,
+                            maxLines: 1,
+                            overflow: TextOverflow.ellipsis,
+                            style: const TextStyle(
+                                color: Colors.black,
+                                fontSize: 20,
+                                fontWeight: FontWeight.bold)),
+                        const SizedBox(height: 8),
+                        Text(todoCard.description,
+                            maxLines: 2,
+                            overflow: TextOverflow.ellipsis,
+                            style: const TextStyle(fontSize: 14)),
+                        const SizedBox(height: 24),
+
+                        //TODO: get like todoCard.createdDate using timestamp
+                        Text(getTodoCardDisplayDate(DateTime.now()),
+                            maxLines: 1,
+                            overflow: TextOverflow.ellipsis,
+                            style: const TextStyle(fontSize: 14)),
+                      ],
+                    ),
+                  ),
+                ),
+                Padding(
+                  padding: const EdgeInsets.only(right: 8, top: 8),
+                  child: InkWell(
+                    onLongPress: () => {
+                      //TODO: show toast
+                      //Pin the card to top
+                    },
+                    child: ImageIcon(
+                      AssetImage(AssetsManager.icons.pinIcon),
+                      color: Colors
+                          .red, //TODO: change the colour based on pinned or not
+                      size: 20,
```

Comment:
@dilumdesilva To avoid hard coding and refer to ui manager classes. Better to use styles.